client/state/action-types.ts

```diff
@@ -946,6 +946,7 @@ export const SITE_SETTINGS_SAVE_FAILURE = 'SITE_SETTINGS_SAVE_FAILURE';
 export const SITE_SETTINGS_SAVE_SUCCESS = 'SITE_SETTINGS_SAVE_SUCCESS';
 export const SITE_SETTINGS_UPDATE = 'SITE_SETTINGS_UPDATE';
 export const SITE_STATS_RECEIVE = 'SITE_STATS_RECEIVE';
+export const ALL_SITES_STATS_RECEIVE = 'ALL_SITES_STATS_RECEIVE';
```

Comment:
Nit: action types in this file are sorted in alphabetic order. I'd recommend moving this to the top.